build-logic/src/main/kotlin/Node.kt

```diff
@@ -17,10 +17,8 @@ fun Project.configureNode() {
     )
   }
 
-  // See https://youtrack.jetbrains.com/issue/KT-47215
```

Comment:
`disableGranularWorkspaces()` no longer exists, but it looks like the original issue KT-47215 is also no longer relevant

Reply:
Using a dev version of the RC fixes it.